src/test/java/org/mockitousage/bugs/varargs/VarargsNotPlayingWithAnyObjectTest.java

```diff
@@ -35,18 +37,6 @@ public void shouldMatchAnyVararg() {
         verify(mock, never()).run(anyString(), eq("f"));
     }
 
-    //we cannot use anyObject() for entire varargs because it makes the verification pick up extra invocations
```

Comment:
Are you removing the test because errorprone does not like empty catch statement? Hmm, it is not a good (even though I love removing code :). If you propose to remove it, please validate carefully and ensure we're not reducing coverage.

From my initial look, the test is valid, even though it covers a corner case. I suggest to either suppress the warning or use AssertJ's assertThrownBy

Reply:
It was because we were actually not asserting anything in this case. The reason is that the `fail()` statement was always executed, because `verify(mock).run((String[]) anyObject());` is completely valid and works. This would then throw an `AssertionError`, which is consequently caught in the `try-catch` in the test case. In other words, whether the `fail` is reached or not, we would always catch the error and the test would never fail (even if there is a bug). Errorprone is correct in identifying that this is an issue.

Reply:
@mockitoguy Any update regarding this comment?

Reply:
Oh I see! Thank you for clarification!

In that case we should absolutely clean up the test method. I'll push a commit with a desired fix and merge the PR.

Thank you for digging deep into this one. I missed the flaw in this test. Clean tests for the win!!!